loops.js

```diff
@@ -29,6 +37,14 @@
 // 3 is oneven
 // .... etc.
 // 15 is oneven
+for (let i = 0;  i <15;  i++) {
```

Comment:
Je zet in deze regel i <15, waardoor "15 is oneven" niet gelogd wordt.
Schoonheidsfoutje: geen spatie tussen < en 15.